samples-final/AspNetCore-LUIS-Bot/LuisBot.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Bot;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Core.Extensions;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Builder.Ai.LUIS;
+using Microsoft.Bot.Builder.Dialogs;
+using System;
+using System.Collections.Generic;
+using Prompts = Microsoft.Bot.Builder.Prompts;
+using Newtonsoft.Json.Linq;
+using Microsoft.Recognizers.Text;
+using System.Linq;
+
+namespace AspNetCore_LUIS_Bot
+{
+    public class LuisBot : IBot
+    {
+        private const double LUIS_INTENT_THRESHOLD = 0.2d;
+
+        private readonly DialogSet dialogs;
+
+        public LuisBot()
+        {
+            dialogs = new DialogSet();
+            dialogs.Add("Fallback", new WaterfallStep[] { DefaultDialog });
+            dialogs.Add("None", new WaterfallStep[] { DefaultDialog });
+            dialogs.Add("Calendar.Add", new WaterfallStep[] { AskRemainderTitle, AskRemainderTime, AskRemainderConfirmation });
```

Comment:
This is great ,but probably more complex than we want in this "Simple" sample. 

Let's remove some of the scenarios to simplify.